test/commands/add/tag_spec.rb

```diff
@@ -29,35 +29,42 @@
 
   describe "when friend name has one match" do
     let(:friend_name) { "George" }
-    let(:tag) { "school" }
+    let(:tag) { "school:tuskegee-institute" }
 
     it "adds tag to friend" do
-      line_changed "- George Washington Carver", "- George Washington Carver @school"
+      line_changed(
+        "- George Washington Carver",
+        "- George Washington Carver @school:tuskegee-institute"
+      )
     end
 
     it "prints an output message" do
-      stdout_only 'Tag added to friend: "George Washington Carver @school"'
+      stdout_only 'Tag added to friend: "George Washington Carver @school:tuskegee-institute"'
     end
 
     describe "when tag is passed with @" do
-      let(:tag) { "@school" }
+      let(:tag) { "@school:tuskegee-institute" }
 
       it "adds tag to friend" do
-        line_changed "- George Washington Carver", "- George Washington Carver @school"
+        line_changed(
+          "- George Washington Carver",
+          "- George Washington Carver @school:tuskegee-institute"
+        )
       end
 
       it "prints an output message" do
-        stdout_only 'Tag added to friend: "George Washington Carver @school"'
+        stdout_only 'Tag added to friend: "George Washington Carver @school:tuskegee-institute"'
       end
     end
 
     describe "when friend already has tags" do
       let(:friend_name) { "Marie" }
+      let(:tag) { "school:ecole-normale-superieure" }
```

Comment:
Very minor, but would you mind adding the French accents here? (`school:école-normale-supérieure`) I think Ruby/the shell/the regex should all handle those characters fine but it would be nice to have it in a test.

Reply:
Should we do that, though? If we encourage non-7-bit-ASCII tags, we ought to do some sort of normalization. I had a look at [the wikipedia page](https://en.wikipedia.org/wiki/Unicode_equivalence) on the subject, looked at NFD, NFC, NFKD, and NFKC and the best method I have at hand for picking the best normalization form for our purposes is eenie-meenie-miney-moe.

Reply:
I don't feel strongly here so I'll defer to you. Thanks for doing so much digging on this!